ge05_team4/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "ge05_team04",
+    "summary": "Distance based warehouse computation on SO",
+    "description": """Changed Sales Orders so that the nearest existing warehouse
+    to the specified delivery order is assigned to it automaticaly. 
+    Task Id:3427342 Dev: Odoo Team 4""",
+   'license': 'OPL-1',
+    "version": "1",
```

Comment:
Use Semantic Versioning
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes(I have never seen this number modified in odoo hehe)
MINOR version when you add functionality in a backward compatible manner(In odoo we increment this number when we upgrade code form versions)
PATCH version when you make backward compatible bug fixes(bug fixes in odoo as well)

So in this case, since the module is new version should be 1.0.0, but it's important to know when we increased it